connector.js

```diff
@@ -9,125 +11,116 @@
  * @class HttpConnector
  */
 
-import AWS from 'aws-sdk';
-import HttpConnector from 'elasticsearch/src/lib/connectors/http'
-import _ from 'elasticsearch/src/lib/utils';
-import zlib from 'zlib';
+const AWS = require('aws-sdk');
+const HttpConnector = require('elasticsearch/src/lib/connectors/http');
+const zlib = require('zlib');
 
 class HttpAmazonESConnector extends HttpConnector {
   constructor(host, config) {
     super(host, config);
-    const { protocol, port } = host;
+
+    const protocol = host.protocol;
+    const port = host.port;
     const endpoint = new AWS.Endpoint(host.host);
 
     if (protocol) endpoint.protocol = protocol.replace(/:?$/, ":");
     if (port) endpoint.port = port;
 
-    this.AWS = AWS;
     this.awsConfig = config.awsConfig || AWS.config;
     this.endpoint = endpoint;
     this.httpOptions = config.httpOptions || this.awsConfig.httpOptions;
+    this.httpClient = new AWS.NodeHttpClient();
```

Comment:
it seemed unnecessary to create the http *client* on every request. thoughts?

Reply:
I left it here since it was a bit easier to set up tests that way; I added a few tests.